### DISCUSSION TITLE
[SQL Import] Excluding columns from embedded tables

### DISCUSSION THREAD

**Discussion by Shinyshark1** [ravendb/ravendb#16682] [Q&A] [answered]
I'm working on importing my SQL Server data into a RavenDB. I'm using the migration tool that RavenDB provides. I want to embed tables, but not all of their columns. For example, I have a table with users that has a lot of information that should be visible on the user detail page. I want to embed parts of the user into my posts, so that I may show their name and link to the row in the users table. I was unable to find out how I can specify that I want to exclude certain columns in the embed. I tried removing their names to indicate I did not want them, but RavenDB notifies me that columns now have duplicate names.

Is it possible to exclude columns at all in the RavenDB import? I'm ok with a workaround as well if anyone knows of it.

**Comment by ayende** [comment]
The easiest way to do this is to do the filter in the JS script.
See the section on transforming documents:
https://ravendb.net/docs/article-page/5.4/csharp/studio/database/tasks/import-data/import-from-sql#filtering-/-transforming-documents

**Reply by Shinyshark1** [reply]
So would I just set the values to be string.Empty or null? I want to omit them entirely but if I skip, I skip the whole row.

**Reply by ayende** [reply, marked as answer]
You need to post the entire thing. Really hard to understand otherwise.

At any rate,  you can `delete` fields that you don't care about.

**Reply by Shinyshark1** [reply]
Ok thank you, that's what I was going for with my current migration plan.
